packages/wrapped-keys/src/lib/lit-actions-client/utils.ts

```diff
@@ -48,3 +49,47 @@ export function postLitActionValidation(
 export function getLitActionCid(network: Network, actionType: LitActionType) {
   return LIT_ACTION_CID_REPOSITORY[actionType][network];
 }
+
+/**
+ * Fetches and updates the params.code if it matches the given IPFS ID, and
+ * set the IPFS ID to undefined.
+ *
+ * @param params - The JSON execution SDK parameters.
+ * @returns A promise that resolves to the updated JSON execution SDK parameters.
+ * @throws If there is an error fetching the code or if the response is not successful.
+ */
+export async function fetchAndUpdateCodeIfMatch(
+  params: JsonExecutionSdkParams
+): Promise<JsonExecutionSdkParams> {
+  for (const [action, platforms] of Object.entries(LIT_ACTION_CID_REPOSITORY)) {
+    for (const [platform, cid] of Object.entries(
+      platforms as Record<string, string>
+    )) {
+      if (cid === params.ipfsId) {
```

Comment:
is there case where `cid` will not match?

Reply:
yea - if they are not wrapped key lit actions, then it will just return the default params